test/common/src/main/kotlin/org/codice/compliance/verification/binding/PostBindingVerifier.kt

```diff
@@ -39,12 +41,28 @@ class PostBindingVerifier(private val response: IdpPostResponseDecorator) {
      * Verify the response for a post binding
      */
     fun verify() {
+        verifyHttpStatusCode()
         decodeAndVerify()
         verifySsoPost()
         verifyPostRelayState()
         verifyPostDestination()
     }
 
+    /**
+     * Verifies the http status code of the response according to the post binding rules in the post spec
+     * 3.5.6 Error Reporting
+     */
+    private fun verifyHttpStatusCode() {
+        if (response.httpStatusCode != 200) {
+            SAMLComplianceException.createWithPropertyNotEqualMessage(
```

Comment:
✏️ Because I expect my exception PR - #34 - will be merged first, you'll need to rebase and make a very small change here.